target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -8,6 +8,11 @@ board=$(board_name)
 boardname="${board##*,}"
 
 case "$board" in
+airtight,c-65)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
+	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:green:wlan2g" "phy1tpt"
```

Comment:
Drop the wlan2g/wlan5g lines and use `linux,default-trigger = "phy0tpt";` and `linux,default-trigger = "phy1tpt";` in your LED nodes instead. Configuration of LEDs in DTS is preferred. Example:
https://github.com/openwrt/openwrt/blob/e65f935de9e67ea07a251f114455f12e9555c5bf/target/linux/ath79/dts/ar9344_comfast_cf-e120a-v3.dts#L36-L40

Reply:
If there is no GPIO from SoC controlling the wlan5g led then it should stay defined in this file.

Reply:
Indeed no GPIO on the SoC, it is on the WLAN card.